javav2/example_code/kendra/src/test/java/KendraTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import com.example.kendra.*;
+import org.junit.jupiter.api.*;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.kendra.KendraClient;
+
+import java.io.*;
+import java.util.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+
+public class KendraTest {
+
+    private static KendraClient kendra;
+    private static String indexName = "";
+    private static String indexDescription = "";
+    private static String indexRoleArn = "";
+    private static String indexId = "";
+    private static String s3BucketName = "";
+    private static String dataSourceName = "";
+    private static String dataSourceDescription = "";
+    private static String dataSourceRoleArn = "";
+    private static String dataSourceId = "";
+    private static String text = "";
+
+    @BeforeAll
+    public static void setUp() throws IOException {
+
+        // Run tests on Real AWS Resources
+        Region region = Region.US_EAST_1;
+        kendra = KendraClient.builder().region(region).build();
+
+        try (InputStream input = KendraTest.class.getClassLoader().getResourceAsStream("config.properties")) {
+
+            Properties prop = new Properties();
+
+            if (input == null) {
+                System.out.println("Sorry, unable to find config.properties");
+                return;
+            }
+
+            //load a properties file from class path, inside static method
```

Comment:
// Load a properties file from the class path, inside static method.

Reply:
fixed